### PR TITLE
feat(battles): advanced search tab

### PR DESCRIPTION
**Pull request by j-masci**

### CODE REVIEW

**Comment by j-masci** [comment]
Added search tab to battles page...

Possible future additions not included here:
- add cripple selector
- use kuski select field similar to the one on chat log
- could scrap the submit button and debounce the inputs instead (not so sure this would really accomplish much)

image

File: src/components/Inputs.js [hunk 1]
```diff
@@ -14,10 +14,17 @@ const Container = styled.div`
   margin-right: 10px;
 `;
 
-export const Dropdown = ({ name, options, selected, update }) => {
+export const Dropdown = ({
+  name,
+  options,
+  selected,
+  update,
+  className,
```
Comment (sunehs) [inline src/components/Inputs.js:22]: What are you using className for? shouldn't really be needed with styled components and I don't see the param used anywhere in this pr

Comment (j-masci) [reply src/components/Inputs.js:22]: I think that has to be done so that styled components can extend the component. ie. styled(Input) or something

Comment (j-masci) [reply src/components/Inputs.js:22]: it's possible that the input isn't being extended in this way. It was a while ago now so I don't recall. I just know when I put a bunch of form elements in a row they never quite line up so I have to do a lot of adding some margin top/bottom to different things. So I'm guessing that's why the class name was there, but not sure if its necessary at the moment.

**Pull request merged**